freqtrade/rpc/api_server/webserver.py

```diff
@@ -210,6 +211,9 @@ async def _broadcast_queue_data(self):
             # Disconnect channels and stop the loop on cancel
             await self._ws_channel_manager.disconnect_all()
             self._ws_loop.stop()
+            # Avoid adding more items to the queue if they aren't
+            # going to get broadcasted.
+            self._ws_queue = None
```

Comment:
This is the killer feature - as it'll mean that even if this loop crashes for some reason - it'll never start to leak due to queue getting full.